packages/core/ios/Sources/DdSdkConfiguration.swift

```diff
@@ -31,26 +33,26 @@ import Foundation
 public class DdSdkConfiguration: NSObject {
     public var clientToken: String = ""
     public var env: String = ""
-    public var applicationId: String? = nil
+    public var applicationId: String = ""
     public var nativeCrashReportEnabled: Bool? = nil
     public var nativeLongTaskThresholdMs: Double? = nil
     public var longTaskThresholdMs: Double = 0.0
     public var sampleRate: Double? = nil
-    public var site: NSString? = nil
+    public var site: DatadogSite
     public var trackingConsent: NSString? = nil
     public var telemetrySampleRate: Double? = nil
     public var vitalsUpdateFrequency: NSString? = nil
     public var trackFrustrations: Bool? = nil
-    public var uploadFrequency: NSString? = nil
-    public var batchSize: NSString? = nil
+    public var uploadFrequency: Datadog.Configuration.UploadFrequency
+    public var batchSize: Datadog.Configuration.BatchSize
```

Comment:
I believe these defaults dont make any sense since we always set these values in the `init` function.
Actually we should probably remove all of them, as they could be misleading and this would allow us to detect if any value is not set by accident in the `init` function.
Do you agree or do you see a good reason to keep them?